generated/nirfmxwcdma/nirfmxwcdma.proto

```diff
@@ -13,53 +13,3435 @@ option csharp_namespace = "NationalInstruments.Grpc.NiRFmxWCDMA";
 
 package nirfmxwcdma_grpc;
 
+import "nidevice.proto";
 import "session.proto";
 
 service NiRFmxWCDMA {
```

Comment:
Didn't have a good spot to comment but wanted to make sure that there's separate follow up work to expand on the system tests to spot check various areas.

Reply:
There is a task (AB#2280303) to add system-level spot tests, and @zhqinbo is already working on it.